### PR TITLE
Update Developer Guide for v1.1

### PR DESCRIPTION
**Pull request by JunWei3112**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -357,6 +357,14 @@ Preconditions: The student's details have been previously added to TutorAid.
     
 **Extensions**
 
+* 1a. TutorAid detects an error in the list command.
+
+    * 1a1. TutorAid displays an error message and requests the tutor to re-enter the command.
+    * 1a2. User re-enters the command.
+
+      Steps 1a1-1a2 are repeated until the data entered are correct.
```
Comment (Preshita01) [inline docs/DeveloperGuide.md:365]: Maybe can rephrase this to 

`Steps 1a1-1a2 are repeated until the command entered are correct.` 

so that it better suits the use case. This can also be applied to some of use cases below.

Otherwise LGTM

Comment (JunWei3112) [reply docs/DeveloperGuide.md]: Fixed, thanks!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -357,6 +357,14 @@ Preconditions: The student's details have been previously added to TutorAid.
     
 **Extensions**
```
Comment (Preshita01) [inline docs/DeveloperGuide.md:358]: LGTM

**Review by Preshita01 — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (dt-td) [reply docs/DeveloperGuide.md:358]: LGTM

**Review by dt-td — APPROVED** [review]
